public/pages/Rollups/containers/Rollups/Rollups.tsx

```diff
@@ -0,0 +1,583 @@
+        }
+      } catch (err) {
+        toastNotifications.addDanger(getErrorMessage(err, "Could not delete the rollup job"));
+      }
+    }
+  };
+
+  onDisable = async (): Promise<void> => {
+    const { rollupService } = this.props;
+    const { selectedItems } = this.state;
+    var item;
+    for (item of selectedItems) {
+      const rollupId = item._id;
+      try {
+        const response = await rollupService.stopRollup(rollupId);
+
+        if (response.ok) {
+          await this.getRollups();
+          await this.getExplains();
+          //Show success message
+          toastNotifications.addSuccess(`${rollupId} is disabled`);
+        } else {
+          toastNotifications.addDanger(`Could not stop the rollup job "${rollupId}" : ${response.error}`);
+        }
+      } catch (err) {
+        toastNotifications.addDanger(getErrorMessage(err, "Could not stop the rollup job"));
+      }
+    }
+  };
+
+  onEnable = async (): Promise<void> => {
+    const { rollupService } = this.props;
+    const { selectedItems } = this.state;
+    var item;
+    for (item of selectedItems) {
+      const rollupId = item._id;
+      try {
+        const response = await rollupService.startRollup(rollupId);
+
+        if (response.ok) {
+          await this.getRollups();
+          await this.getExplains();
+          //Show success message
+          toastNotifications.addSuccess(`${rollupId} is enabled`);
+        } else {
+          toastNotifications.addDanger(`Could not start the rollup job "${rollupId}" : ${response.error}`);
+        }
+      } catch (err) {
+        toastNotifications.addDanger(getErrorMessage(err, "Could not start the rollup job"));
+      }
+    }
+  };
+
+  onTableChange = ({ page: tablePage, sort }: Criteria<ManagedCatIndex>): void => {
+    const { index: page, size } = tablePage;
+    const { field: sortField, direction: sortDirection } = sort;
+    this.setState({ from: page * size, size, sortField, sortDirection });
+  };
+
+  onSelectionChange = (selectedItems: RollupItem[]): void => {
+    this.setState({ selectedItems });
+  };
+
+  onSearchChange = (e: React.ChangeEvent<HTMLInputElement>): void => {
+    this.setState({ from: 0, search: e.target.value });
+  };
+
+  onPageClick = (page: number): void => {
+    const { size } = this.state;
+    this.setState({ from: page * size });
+  };
+
+  resetFilters = (): void => {
+    this.setState({ search: DEFAULT_QUERY_PARAMS.search });
+  };
+
+  onActionButtonClick = (): void => {
+    this.setState({ isPopoverOpen: !this.state.isPopoverOpen });
+  };
+
+  closePopover = (): void => {
+    this.setState({ isPopoverOpen: false });
+  };
+
+  closeDeleteModal = (): void => {
+    this.setState({ isDeleteModalVisible: false });
+  };
+
+  showDeleteModal = (): void => {
+    this.setState({ isDeleteModalVisible: true });
+  };
+
+  concatName = (): string => {
```

Comment:
Can do something like
```js
getSelectedRollupIds = (): string => this.state.selectedItems.map(item => item._id).concat(", ")
```

Reply:
This would return string[] and become all job names concatenated with " ," at the end. Will look into how to rewrite later.

Reply:
Whoops, change `concat(", ")` to `join(", ")` in the above